Microsoft.Toolkit.Win32.UI.Controls/WinForms/WebView/WebView.cs

```diff
@@ -0,0 +1,456 @@
+// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Drawing;
+using System.Drawing.Drawing2D;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Security.Permissions;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using Microsoft.Toolkit.Win32.UI.Controls.Interop.Win32;
+using Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT;
+using Windows.Web.UI.Interop;
+using WebViewControlDeferredPermissionRequest = Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT.WebViewControlDeferredPermissionRequest;
+using WebViewControlMoveFocusReason = Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT.WebViewControlMoveFocusReason;
+using WebViewControlProcess = Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT.WebViewControlProcess;
+using WebViewControlSettings = Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT.WebViewControlSettings;
+
+namespace Microsoft.Toolkit.Win32.UI.Controls.WinForms
+{
+    /// <summary>
+    /// This class is an implementation of <see cref="IWebView"/> for Windows Forms. This class cannot be inherited.
+    /// </summary>
+    /// <seealso cref="Control" />
+    /// <seealso cref="ISupportInitialize" />
+    [Designer(typeof(WebViewDesigner))]
+    [DefaultProperty(Constants.ComponentDefaultProperty)]
+    [DefaultEvent(Constants.ComponentDefaultEvent)]
+    [Docking(DockingBehavior.AutoDock)]
+    [SecurityCritical]
+    [PermissionSet(SecurityAction.InheritanceDemand, Name = Constants.SecurityPermissionSetName)]
+    public sealed partial class WebView : Control, IWebView, ISupportInitialize
```

Comment:
@ajbennet  Need to define a category (e.g. so it shows up next to WebBrowser)? What about an icon?